packages/tools/webpack-fluid-loader/src/loader.ts

```diff
@@ -30,6 +30,11 @@ import { RequestParser } from "@fluidframework/runtime-utils";
 import { MultiUrlResolver } from "./multiResolver";
 import { deltaConns, getDocumentServiceFactory } from "./multiDocumentServiceFactory";
 
+// SetImmediate is needed by broadcaster/lambdas, but is not available in the browser.
+// Need a polyfill until this issue is resolved https://github.com/microsoft/FluidFramework/issues/4040
+// eslint-disable-next-line import/no-unassigned-import
+import "setimmediate";
```

Comment:
I am confused. How does this help the isomorphicity of lambdas? Is it only for local-server?

Reply:
So, set immediate is not available on modern browsers. When I set the webpack config for webpack-fluid-loader to have the value node: false, this makes it such that webpack-fluid-loader does not include and node polyfills. Doing so would be a build error, which is a good way to make sure we don't regress here.

In terms of why we need this change, this is the last node polyfill we need, so I'm providing it manually until #4040 is resolved. With the move the node: false, coauthoring with local server would fail silently due to the lack of set immediate.

Reply:
Makes sense. So this is not an immediate blocker. Cause we are only allowing one exception, everything else will still be blocked.